### PR TITLE
Update FamBPMs device

### PR DESCRIPTION
**Pull request by anacso17**
This PR updates FamBPMs device:
- remove wait methods that use orbit comparison
- add methods to verify that data was updated using PVs timestamp
- fix docstrings and implement some linting suggestions

Maybe the changes here will trigger changes in `apsuite`, I will verify it, but if you guys notice some undesired API change, please let me know.

### CODE REVIEW

File: siriuspy/siriuspy/devices/bpm.py [hunk 1]
```diff
@@ -1289,14 +1327,51 @@ def mturn_wait_update_orbit(self, timeout=10, consider_sum=False) -> int:
...
+        Args:
+            timeout (int, optional): Waiting timeout. Defaults to 10.
+            consider_sum (bool, optional): Whether to also wait for sum signal
+                to be updated. Defaults to False.
+
+        Returns:
+            int: code describing what happened:
+                -1: initial timestamps were not defined;
+                =0: data updated.
+                >0: index of the first FOFB controller which did not update
```
Comment (fernandohds564) [inline siriuspy/siriuspy/devices/bpm.py:1354]: `FOFB controller`.

**Review by fernandohds564 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

File: siriuspy/siriuspy/devices/bpm.py [hunk 2]
```diff
@@ -1351,7 +1290,7 @@ def mturn_wait_update_timestamps(
             int: code describing what happened:
                 -1: initial timestamps were not defined;
                 =0: data updated.
-                >0: index of the first FOFB controller which did not update
+                >0: index of the first BPM which did not update
                     plus 1.
```
Comment (fernandohds564) [inline siriuspy/siriuspy/devices/bpm.py:1294]: I think this text fits in the previous line, doesn't it?

**Review by fernandohds564 — APPROVED** [review]

File: siriuspy/siriuspy/devices/bpm.py (hunk 2 above)
Comment (anacso17) [reply siriuspy/siriuspy/devices/bpm.py]: done!

File: siriuspy/siriuspy/devices/bpm.py [hunk 3]
```diff
@@ -1207,19 +1237,20 @@ def set_switching_mode(self, mode='direct'):
...
+        self._initial_timestamps = self.get_mturn_timestamps(
+            return_sum=consider_sum)
 
     def mturn_reset_flags(self):
         """Reset Multiturn flags to wait for a new orbit update."""
         for flag in self._mturn_flags.values():
             flag.clear()
 
-    def mturn_reset_flags_and_update_initial_orbit(self, consider_sum=False):
+    def mturn_reset_flags_and_update_initial_times(self, consider_sum=False):
```
Comment (fernandohds564) [inline siriuspy/siriuspy/devices/bpm.py:1250]: I think you could call this method: `mturn_reset_flags_and_update_initial_timestamps`.

Comment (anacso17) [reply siriuspy/siriuspy/devices/bpm.py:1250]: done!

**Pull request merged**